index.html

```diff
@@ -282,17 +267,20 @@
    if (!error) { // error = 0 => everything okay
       if ('cover' in data) { // data is dict, should have a key cover
         var cover = data.cover;
-        console.log('cover', cover);      
+        console.log('cover', cover);
         d3.selectAll('.link') // execute function on each edge
         .each(function(d) {
                   var header = d3.select(this);
                   // Convert to array with the index as the number corresponding to a vertex
                   // (this also adds links (1,1), (2,2) etc but this wont affect it, as they will not be in cover)
                   var d_arr = Object.keys(d).map(function (key) {return Number(d[key].index)});
-                
+
                   if (inArray(cover, d_arr)) { // if edge is in cover color it red
                       header.style({'stroke': 'red'});
                       header.style({'stroke-width': 3});
+                  } else {
+                      header.style({'stroke': '#999'});
+                      header.style({'stroke-width': 1});
```

Comment:
We need to be sure to recolor edges that aren't in the cover gray. That way if the user clicks 'Compute Edge Cover' twice only those edges in the second cover will be colored red.